pkgs/top-level/all-packages.nix

```diff
@@ -31948,7 +31948,9 @@ with pkgs;
 
   mpvScripts = mpv-unwrapped.scripts;
 
-  shaka-packager = callPackage ../applications/video/shaka-packager { };
+  shaka-packager = callPackage ../by-name/sh/shaka-packager/package.nix {
+    abseil-cpp = abseil-cpp_202401;
```

Comment:
We do not explicitly `callPackage` derivations in the `pkgs/by-name` directory. Please remove this and pass `abseil-cpp_202401` directly into your derivation.

Reply:
Disregard, `pkgs/by-name` is accepting of this due to the requirement you have.